src/Presentation/Presenters/DonationConfirmationHtmlPresenter.php

```diff
@@ -19,13 +19,14 @@ public function __construct( TwigTemplate $template ) {
 		$this->template = $template;
 	}
 
-	public function present( Donation $donation, string $updateToken ): string {
-		return $this->template->render( $this->getConfirmationPageArguments( $donation, $updateToken ) );
+	public function present( Donation $donation, string $updateToken, string $templateName = '' ): string {
```

Comment:
MediaWiki and Twig don't work together that well in this particular case:

The content in MediaWiki needs to be enclosed within `<html>` to prevent `{% extends 'BaseLayout.twig' %}` from being url encoded as `{% extends 'BaseLayout.twig'&#160;%}`, which makes Twig complain about an unexpected `&`.

The MW parser at least encloses the content within a `<p />` tag which makes Twig complain about content being outside of blocks.